Documentation/Configuration/ConfigurationFiles.rst

```diff
@@ -0,0 +1,26 @@
+.. include:: /Includes.rst.txt
+
+.. _configuration-files:
+
+===================
+Configuration files
+===================
+
+The configuration files :file:`LocalConfiguration.php` and
+:file:`AdditionalConfiguration.php` are located in the directory
+:ref:`public/typo3conf/ <directory-public-typo3conf>` in Composer-based
+installations. In legacy installations they are located in
+:ref:`typo3conf/ <legacy-directory-typo3conf>`.
+
+The most important configurations file is
+:file:`LocalConfiguration.php`. It contains local settings of the
+main global PHP array :php:`$GLOBALS['TYPO3_CONF_VARS`], crucial settings
+like database connect credentials are in here. The file is managed by the
+:guilabel:`Admin Tools`.
+
+The file :file:`LocalConfiguration.php` can be enriched by
```

Comment:
How can a file be enriched by another file? Maybe better: "The configuration can be adjusted by AdditionalConfiguration" or something similar.

Reply:
The configuration can be overridden by settings in the file AdditionalConfiguration.php. Despite some downsides this is a widely used option as it allows easily different configuration for various environments like production and development without changing any file: the file LocalConfiguration.php is used for production and developers can create their own configuration in AdditionalConfiguration.php.